testing/gn/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor:
+# Maintainer: Sven Assmann <sven@d34dl0ck.me>
+pkgname=gn
+pkgver=1530
+pkgrel=1ab6fa2c
+pkgdesc="GN is a meta-build system that generates build files for Ninja."
+url="https://gn.googlesource.com/gn/+/refs/heads/master/README.md"
+arch="all"
+license="BSD-3-Clause"
+depends="libgcc musl"
+makedepends="python2 ninja git clang"
+# source="$pkgname-$pkgver.tar.gz::https://gn.googlesource.com/gn/+archive/refs/heads/master.tar.gz"
+giturl="https://gn.googlesource.com/gn"
+builddir="$srcdir"
+
+fetch() {
+    git clone https://gn.googlesource.com/gn "$builddir/git"
```

Comment:
This is not deterministic. Whenever they push commits to master, it would use a different commit. What is the reason you don't use something like https://gn.googlesource.com/gn/+archive/1ab6fa2cab7ec64840db720a56018ca8939329f9.tar.gz as the source so that you know you get the same source every time?

Reply:
well the tar.gz cannot be used, because their build script needs git, so that it can generate a header file that contains the commit hash that it was build on. 
But as said above, I will make sure I'll pin the commit hash.